webpack.prod.config.js

```diff
@@ -0,0 +1,88 @@
+var Path = require('path');
```

Comment:
I think it would make more sense to use a _single_ `webpack.config.js` that is dynamic. We already set `NODE_ENV=production` before invoking webpack for the minified build. I would prefer if you make the single `webpack.config.js` look at `process.env.NODE_ENV` to include / exclude the production differences.

Do you see any reasons why this would be a bad idea?